.github/workflows/test.yml

```diff
@@ -24,8 +24,19 @@ jobs:
             vcvars: "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Enterprise\\VC\\Auxiliary\\Build\\vcvars64.bat"
           }
         - {
-            os: ubuntu-20.04
+            os: ubuntu-20.04,
+            sanitizers: ON
           }
+        - {
+            os: macos-10.15,
+            # asan needs the pre-load trick for python, ubsan seems to
+            # have false positives:
+            #   InfoDictionaryTest.cpp:346:25: runtime error:
+            #      call to function openassetio_v0_0_InfoDictionary_getBool
+            #      through pointer to incorrect function type
+            sanitizers: OFF
+          }
```

Comment:
[question]] What do you think of the pseudo-JSON syntax (the `{ ... }`s and trailing `,`s)? I'm not a big fan in this case, it just seems to add unnecessary verbosity.

Reply:
With you on that, updated [here](https://github.com/OpenAssetIO/OpenAssetIO/pull/382/commits/2126aa6f873f184683486b46506798ec3fac4f6b).